stable/sonarqube/templates/service.yaml

```diff
@@ -7,6 +7,10 @@ metadata:
     chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+{{- if .Values.service.annotations }}
```

Comment:
You'll have to add the key to values.yaml as well as providing some documentation about it in README.